app/models/question.rb

```diff
@@ -0,0 +1,45 @@
+class Question < ApplicationRecord
+  include Swagger::Blocks
+
+  belongs_to :assessment
+  belongs_to :format
+  belongs_to :created_by, class_name: 'User'
+  has_many :solutions, inverse_of: :question
+  has_many :assets, as: :owner
```

Comment:
We probably need to think about the sharing of assets.  Possible that we could get 10000 variants of a generative question and they'd all have the same asset.  We probably don't want to store a 1MB JPG 10000 times.  Ross has experience and thoughts in this area.  Dante probably too.

Reply:
yes, let's discuss this.  Dante mentioned that he thought we shouldn't store the assets directly, instead we'd rely on the members to store them and have urls in their content.  Not sure if that's  a good idea or not.